diktat-analysis.yml

```diff
@@ -107,8 +107,10 @@
   configuration:
     valueArgument: true
     valueParameter: true
+# reduces speed of development on the FE
+# will remove it by for now
 - name: MAGIC_NUMBER
-  enabled: true
+  enabled: false
```

Comment:
there is no any configuration, such we would disable rule only on frontend paths? `save-frontend/*`?

Reply:
we can put `diktat-analysis.yml` inside to save-frontend. Will test it after splitting

Reply:
actully, it doesn't work: we configure diktat in extension by the following:
```kotlin
diktat {
    diktatConfigFile = rootProject.file("diktat-analysis.yml")
}
```

Looks like we need to modify this configuration to something like:
```kotlin
diktatConfigFile = project.file("diktat-analysis.yml").takeIf { it.exists() } ?: rootProject.file("diktat-analysis.yml")
```